algo/src/test/java/com/hartwig/serve/sources/ckb/region/CkbRegionFileTest.java

```diff
@@ -0,0 +1,36 @@
+package com.hartwig.serve.sources.ckb.region;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Set;
+
+import com.google.common.io.Resources;
+import com.hartwig.serve.ckb.datamodel.clinicaltrial.ImmutableLocation;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.junit.Test;
+
+public class CkbRegionFileTest {
+
+    @Test
+    public void shouldLoadFileWithAndWithoutStates() throws Exception {
+        Set<CkbRegion> regions = CkbRegionFile.read(Resources.getResource("ckb_region/ckb_regions.tsv").getPath());
+        CkbRegion netherlands = regions.stream().filter(r -> r.country().equals("netherlands")).findFirst().orElseThrow();
+        CkbRegion us = regions.stream().filter(r -> r.country().equals("united states")).findFirst().orElseThrow();
+
+        assertTrue(netherlands.includes(location("Netherlands", "Nord Holland")));
```

Comment:
Fairly minor but at the same time super important: Correct spelling is "Noord Holland" :)